src/v2/Components/SavedSearchAlert/Components/__tests__/SavedSearchAlertArtworkGridFilterPills.jest.tsx

```diff
@@ -78,4 +80,70 @@ describe("SavedSearchAlertArtworkGridFilterPills", () => {
 
     expect(screen.getByText("Banksy")).toBeInTheDocument()
   })
+
+  it("display followed artist pills when 'Artists I Follow' is selected", () => {
```

Comment:
**praise:** thanks for the consistent great test cases as always!

Reply:
Thanks!